exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
+const names = "Morteza   ";
+const nameLength = (names.trim()).length;
```

Comment:
nice combining of these 2 methods in 1 line 🙂

Reply:
Thanks